skeleton/cli-bundler/package.json

```diff
@@ -25,40 +24,39 @@
     "gulp-cache": "",
     "gulp-eslint": "",
     // @endif
-
     // @if feat.typescript
     "gulp-typescript": "",
     "gulp-tslint": "",
     "tslint": "",
     "typescript": "",
     "@types/node": "",
     // @endif
-
     // @if feat.less
     "gulp-less": "",
     // @endif
-
-    // @if feat['postcss-basic'] || feat['postcss-typical']
+    // @if feat['postcss-basic'] || feat['postcss-typical'] || feat['postcss-tailwind']
     "gulp-postcss": "",
     "autoprefixer": "",
     // @endif
-    // @if feat['postcss-typical']
+    // @if feat['postcss-typical'] || feat['postcss-tailwind']
     "cssnano": "",
     "postcss-url": "",
     // @endif
-
+    // @if feat['postcss-tailwind']
+    "tailwindcss": "",
+    // @endif
+    // @if feat['cli-bundler']
```

Comment:
This condition is unnecessary, because the file is inside `skeleton/cli-bundler/...`.
To be precise, gulp-if is only needed when one of less/sass/stylus is in use.